client/src/components/Container/index.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
Remember the <Container> caused an error before? Are you sure you want to add the container in?